libyul/optimiser/Suite.cpp

```diff
@@ -445,7 +483,14 @@ void OptimiserSuite::runSequence(std::vector<string> const& _steps, Block& _ast)
 	{
 		if (m_debug == Debug::PrintStep)
 			cout << "Running " << step << endl;
+#ifdef OUTPUT_PERFORMANCE_METRICS
+		chrono::steady_clock::time_point startTime = chrono::steady_clock::now();
+#endif
 		allSteps().at(step)->run(m_context, _ast);
+#ifdef OUTPUT_PERFORMANCE_METRICS
+		chrono::steady_clock::time_point endTime = chrono::steady_clock::now();
+		m_durationPerStepInMicroseconds[step] += std::chrono::duration_cast<std::chrono::microseconds>(endTime - startTime).count();
```

Comment:
```suggestion
		m_durationPerStepInMicroseconds[step] += chrono::duration_cast<chrono::microseconds>(endTime - startTime).count();
```

Reply:
I'd even go further and add `using std::chrono`